serial/serialposix.py

```diff
@@ -538,6 +538,10 @@ def read(self, size=1):
                     raise SerialException('read failed: {}'.format(e))
             if timeout.expired():
                 break
+            elif self._inter_byte_timeout is not None and self._inter_byte_timeout > 0:
+                #atleast one char received --> overwrite timeout by setting it to inter_byte_timeout
+                timeout = Timeout(self._inter_byte_timeout)
```

Comment:
are you sure this is needed? the `inter_byte_timeout` is actually set on the port using `vtime`, so it should already have elapsed that time and a simple `break` -- `if self._inter_byte_timeout is not None` -- would be enough here.

Reply:
it's been a while since I played around with it but as far I can remember I wasn't able to get the inter_char/inter_byte timeout working on a raspberry pi without my fix - I am pretty sure that I  tried out different values for the timeout (milliseconds, seconds etc as mentioned in #396).

Reply:
> enough her

You **inter_byte_timeout** is useless. It does not worked on any platforms.